app/models/course/experience_points/disbursement.rb

```diff
@@ -69,9 +69,7 @@ def save
   # @param [Hash] attributes Experience points record attributes hash
   # @return [Boolean] True if hash represents a valid points award
   def valid_points_record_attributes?(attibutes)
-    attibutes[:course_user_id].present? &&
-      attibutes[:points_awarded].present? &&
-      attibutes[:points_awarded].to_i != 0
```

Comment:
Shall we retain the validation that `experience_points` should be `>= 0` instead? 

Although I did remember there was some discussions about negative experience points, although I don't think we will support that use case now? Or do we?